pages/_app.tsx

```diff
@@ -30,7 +30,7 @@ const { chains, publicClient, webSocketPublicClient } = configureChains(
 
 const { connectors } = getDefaultWallets({
   appName: "RainbowKit App",
-  projectId: "YOUR_PROJECT_ID",
+  projectId: "bee6af70ea57e6499462532060febf40",
```

Comment:
just to clarify, is it okay to commit `projectId`? Since this repo is public

Reply:
let me make it to env as well